spec/rails_helper.rb

```diff
@@ -8,6 +8,7 @@
 # Prevent database truncation if the environment is production
 abort("The Rails environment is running in production mode!") if Rails.env.production?
 require 'rspec/rails'
+require 'support/factory_bot.rb'
```

Comment:
perfect, it's gonna make our tests more interesting and robust. Very nice addition!